api/v1alpha1/servicebinding_types.go

```diff
@@ -44,6 +44,13 @@ type ServiceBindingSpec struct {
 	// +optional
 	SecretName string `json:"secretName"`
 
+	// SecretKey is used as the key inside the secret to store the credentials
+	// returned by the broker encoded as json. If not specified the credentials
+	// returned by the broker will be used directly as the secrets data, which
+	// can cause problems when using complex data structures.
```

Comment:
can you replace to a more positive phrasing? e.g. "SecretKey is used as the key inside the secret to store the credentials returned by the broker encoded as json to support complex data structures. If not specified the credentials returned by the broker will be used directly as the secrets data."

Reply:
I used your suggestion.